tachyon/math/base/semigroups_unittest.cc

```diff
@@ -109,4 +111,96 @@ TEST(SemigroupsTest, AddOverAddInPlace) {
   static_cast<void>(d);
 }
 
+namespace {
+
+class BatchScalarsTest : public testing::Test {
```

Comment:
```suggestion
class BatchScalarTest : public testing::Test {
```
There are cases for Scalar and Scalars, so maybe just Scalar is a general word that encompasses them all.